cephadm-purge-cluster.yml

```
@@ -30,7 +30,7 @@
       fail:
         msg: |
           You must define a group [admin] in your inventory and add a node where
-          admin keyring is present at /etc/ceph/admin.client.keyring
+          admin keyring is present at /etc/ceph/ceph.admin.client.keyring
```

shouldn't it be `ceph.client.admin.keyring` ?
```suggestion
          admin keyring is present at /etc/ceph/ceph.client.admin.keyring
```

looks like L43 and L48 need to be updated too